src/components/token/utils.ts

```diff
@@ -0,0 +1,68 @@
+import { BitcoinNetwork } from '@rsksmart/rif-wallet-bitcoin'
+
+import { ITokenWithoutLogo } from 'src/redux/slices/balancesSlice/types'
+import { TokenSymbol } from 'src/screens/home/TokenImage'
+
+export type TokenOrBitcoinNetwork = ITokenWithoutLogo | BitcoinNetwork
+
+// default order should be RIF, USDRIF, RBTC, BTC and RDOC
+// other tokens should be sorted alphabetically by symbol
+export const sortTokensBySymbol = (
+  a: TokenOrBitcoinNetwork,
+  b: TokenOrBitcoinNetwork,
+) => {
+  const aSymbol = a.symbol.toUpperCase()
+  const bSymbol = b.symbol.toUpperCase()
+  if (
```

Comment:
This looks gygantic can it be simplified to `tokenSortMap.get`?
For example:
```
const tokenSortAMap = new Map([
[TokenSymbol.RIF,  -1],
[TokenSymbol.TRIF, -1],
etc
])

const tokenSortBMap = new Map([
[TokenSymbol.RIF,  1],
[TokenSymbol.TRIF, 1],
etc
])

```

Reply:
It doesn't work because the order of the ifs is important.